### ISSUE TITLE
Can't Add Website Shortcuts to Unlauncher App List

### ISSUE THREAD

**Issue by JBlocklove** [jkuester/unlauncher#151]
I'm trying to add some website shortcuts to my app list in Unlauncher so I can keep the  web browser hidden and just access the couple of websites which are necessary for me to be able to look at on my phone. To do this I'm trying to use the Hermit browser, which lets you turn websites into "Lite Apps" and add those to the home screen. The issue is that when I try to add one of these apps to my app list in Unlauncher through the "Add to Home Screen" option in Hermit I get the following error:
```
"Settings" doesn't allow adding icons automatically. Use a home screen widget instead.
```
[This response](https://www.reddit.com/r/HermitApp/comments/149rded/hermit_cant_add_lite_app_to_home_screen/) from the Hermit dev team on Reddit seems to indicate that the issue is with the launcher not allowing this. Is there any way to fix this in the settings of Unlauncher?

**Comment by jkuester** [comment]
Yes, this is definitely something I want to get working!  I guess this is probably connected to https://github.com/jkuester/unlauncher/issues/4
